src/transformStateWithClones.js

```diff
@@ -1,13 +1,26 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- *
- * @return {Object[]}
- */
 function transformStateWithClones(state, actions) {
   // write code here
+  const duplicate = { ...state };
+  const array = [];
+
+  for (const action in actions) {
```

Comment:
and wait, actions - is it an object?
i guess it's array - then you should use `for ( ... of ...)`